book/_toc.yml

```diff
@@ -31,6 +31,7 @@
           sections:
             - file: tutorials/sar/sentinel1
             - file: tutorials/sar/uavsar
+            - file: tutorials/microstructure/microstructure-tutorial
```

Comment:
this is currently under the SAR tutorial section, so you'll want to move it up one level:
```yaml
- part: Tutorials
  chapters:
    - file: tutorials/index
      sections:
        - file: tutorials/microstructure/microstructure-tutorial
```

Reply:
done! I think. :) moved the tutorial to its own section.